module/VuFind/src/VuFind/Db/Table/Ratings.php

```diff
@@ -234,4 +234,55 @@ public function getStatistics(): array
         $result = $statement->execute();
         return (array)$result->current();
     }
+
+    /**
+     * Get a paginated result of all ratings
+     * made by the user
```

Comment:
More minor comment standardization.

```suggestion
     * Get a paginated result of all ratings made by the user.
```